packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -169,46 +173,234 @@ export const ShopperBasketsMutations = {
+            response: DataType<Client['updateBillingAddressForBasket']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        },
+        updateCustomerForBasket: (
+            params: Argument<Client['updateCustomerForBasket']>,
+            response: DataType<Client['updateCustomerForBasket']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        },
+        updateItemInBasket: (
+            params: Argument<Client['updateItemInBasket']>,
+            response: DataType<Client['updateItemInBasket']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        },
+        updatePaymentInstrumentInBasket: (
+            params: Argument<Client['updatePaymentInstrumentInBasket']>,
+            response: DataType<Client['updatePaymentInstrumentInBasket']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        },
+        updateShippingAddressForShipment: (
+            params: Argument<Client['updateShippingAddressForShipment']>,
+            response: DataType<Client['updateShippingAddressForShipment']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        },
+        updateShippingMethodForShipment: (
+            params: Argument<Client['updateShippingMethodForShipment']>,
+            response: DataType<Client['updateShippingMethodForShipment']>
+        ): CacheUpdateMatrixElement => {
+            const basketId = params.parameters.basketId
+
+            return {
+                ...updateBasketQuery(basketId),
+                ...invalidateCustomerBasketsQuery(customerId)
+            }
+        }
+    }
+}
 
 /**
  * A hook for performing mutations with the Shopper Baskets API.
  */
 export function useShopperBasketsMutation<Action extends ShopperBasketMutationType>(
     action: Action
 ) {
+    // TODO: where are headers and rawResponse ?
+    const queryClient = useQueryClient()
+
+    const customerId = useCustomerId()
+    const cacheUpdateMatrix = getCacheUpdateMatrix(customerId)
```

Comment:
Minor: rename the `queryKeysMatrix` →  `cacheUpdateMatrix` because I think the latter is clearer on what the query keys are for. The keys are _not_ for the query hooks, but the mutations hooks... specifically for their `onSuccess` callback.